quinn/src/platform/unix.rs

```diff
@@ -257,3 +263,57 @@ fn prepare_msg(
     }
     encoder.finish();
 }
+
+fn prepare_recv(
+    buf: &mut IoSliceMut,
+    name: &mut MaybeUninit<libc::sockaddr_storage>,
+    ctrl: &mut cmsg::Aligned<MaybeUninit<[u8; CMSG_LEN]>>,
+    hdr: &mut libc::msghdr,
+) {
+    hdr.msg_name = name.as_mut_ptr() as _;
+    hdr.msg_namelen = mem::size_of::<libc::sockaddr_storage>() as _;
+    hdr.msg_iov = buf as *mut IoSliceMut as *mut libc::iovec;
+    hdr.msg_iovlen = 1;
```

Comment:
Why is this 1? I had expected `buf.len()`

Reply:
We're using exactly one iovec per datagram in an interface that supports gathering multiple iovecs.

Reply:
Oh right. I now remember I had a misunderstanding about this before